invoicer_project/static/js/clients_list_request.js

```diff
@@ -32,60 +26,42 @@ function createClientListContent(data) {
-                    <p class="list_item_user_icon_initials_text" data-client-id="${clientID}">${clientInitials}</p>
+                    <p class="list_item_user_icon_initials_text" data-element-id="${clientID}">${clientInitials}</p>
                 </div>
                 <div class="col-xxl-3 col-xl-3 col-md-3 col-sm-4 col-5 clickable_item list_item_user_name">
-                    <p class="list_client_username"  data-client-id="${clientID}" >${fullName}</p>
+                    <p class="list_client_username"  data-element-id="${clientID}" >${fullName}</p>
                 </div>
                 <div class="col-xxl-4 col-xl-3 col-md-2 col-1 clickable_item list_item_empty_block"  data-client-id="${clientID}"></div>
                 <div class="col-xxl-2 col-xl-2 col-md-3 col-sm-3 col-5 clickable_item list_item_user_debt">
-                    <p class="list_item_user_debt_text" data-client-id="${clientID}">0$</p>
+                    <p class="list_item_user_debt_text" data-element-id="${clientID}">0$</p>
                 </div>
                 <div class="col-xxl-2 col-xl-3 col-md-3 col-sm-3 col-3 list_item_user_buttons" data-client-id="${clientID}">
-                    <!--TODO MD-FILLED-BUTTON HASN'T BEEN PUBLISHED YET-->
-                    <span class="material-symbols-outlined client-info edit-client" data-client-id="${clientID}" style="font-size:28px;">edit</span>
-                    <span class="material-symbols-outlined client-info delete-client" data-client-id="${clientID}" style="font-size:28px;">delete</span>
-                    <md-checkbox class="delete_clients_checkbox" id="list_item_user_delete" data-client-id="${clientID}"></md-checkbox>
+                    <md-standard-icon-button class="client-info edit-client" data-element-id="${clientID}"><span class="material-symbols-outlined">edit</span></md-standard-icon-button>
+                   <md-standard-icon-button class="client-info delete-client" data-element-id="${clientID}"><span class="material-symbols-outlined">delete</span></md-standard-icon-button>
+                    <md-checkbox class="delete_clients_checkbox" id="list_item_user_delete" data-element-id="${clientID}"></md-checkbox>
                 </div>
             </div>`)
     }
 }
 
-async function obtainNewAccessToken() {
-    let response;
-    const data = {refresh: window.localStorage.getItem('refreshToken')};
-    try {
-        response = await fetch(host + '/user/refresh/', {
-            method: "POST",
-            body: JSON.stringify(data)
-        });
-        const newToken = await response.json();
-        const accessToken = newToken['access'];
-        window.localStorage.setItem('accessToken', accessToken);
-    } catch (error) {
-        console.error(error);
-    }
-    return response.status === 200;
-}
-
 async function addElementsDynamically() {
-    let responseFromServer = await getUserData();
+    let responseFromServer = await getUserData("/clients/client/");
```

Comment:
Are you sure that method getUserData() requires this argument?

Reply:
Done